vip-cache-manager/class-wpcom-vip-cache-manager.php

```diff
@@ -419,24 +440,23 @@ public function execute_purges() {
 			$requests = array_merge( $requests, $this->build_purge_request( $url, 'PURGE' ) );
 		}
 
-		$this->ban_urls = $this->purge_urls = array();
+		$this->ban_urls   = [];
+		$this->purge_urls = [];
 
 		if ( empty( $requests ) ) {
 			return;
 		}
 
-		return $this->curl_multi( $requests );
+		$this->curl_multi( $requests );
 	}
 
-	public function purge_site_cache( $when = null ) {
```

Comment:
SonarLint complained that `$when` is never used.

I have checked that no call site passes parameters to `purge_site_cache` anyway.